### PR TITLE
feat: request-postgres-client

### PR DESCRIPTION
**Pull request by skiv71**
This PR adds a new docker image which provides a job to provision (and drop) a database within an existing postgres container. This is being used to create the container "graph-db", which creates the requred database (per chain) in the local postgres

# Considerations
We did not use https://hub.docker.com/r/andreswebs/postgresql-client because it relies on Kubernetes and kubectl whereas we're using Docker Swarm.

### CODE REVIEW

File: request-postgres/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+complete() {
+    echo "done"
+    [[ $KEEP_ALIVE -eq 1 ]] || exit 0
+    echo "Keep alive active!"
+    while :; do
+        sleep 1
+    done
```
Comment (MantisClone) [inline request-postgres/entrypoint.sh:9]: Under what circumstances would we enable `KEEP_ALIVE`?

File: request-postgres/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+echo_italic() {
+    echo -e "\e[3m${1}\e[0m"
+}
```
Comment (MantisClone) [inline request-postgres/entrypoint.sh:14]: What does this do?

File: request-postgres/.resources/apt.txt
Comment (MantisClone) [inline request-postgres/.resources/apt.txt:1]: Please rename folder to `request-postgres-client`

File: request-postgres/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Request Postgres
```
Comment (MantisClone) [inline request-postgres/README.md:1]:
```suggestion
# Request Postgres Client
```

File: request-postgres/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,36 @@
+FROM debian:12-slim
+
```
Comment (MantisClone) [inline request-postgres/Dockerfile:2]: For security, consider running as a user
```suggestion
USER 1001
```

File: request-postgres/entrypoint.sh (hunk 2 above)
Comment (skiv71) [reply request-postgres/entrypoint.sh:14]: Custom logging syntax

File: request-postgres/entrypoint.sh (hunk 1 above)
Comment (skiv71) [reply request-postgres/entrypoint.sh:9]: This option provides the means to keep a job / container alive so you can execute a shell (if need be) do debug anything. Without it, the container simply exits

File: request-postgres/Dockerfile (hunk 4 above)
Comment (skiv71) [reply request-postgres/Dockerfile]: Not sure what this would achieve?

Comment (MantisClone) [reply request-postgres/Dockerfile]: https://docs.docker.com/develop/develop-images/instructions/#user
> If a service can run without privileges, use USER to change to a non-root user.

It's a security best practice. This same comment applies to all the docker images.

**Review by MantisClone — APPROVED** [review]

File: request-postgres/Dockerfile (hunk 4 above)
Comment (skiv71) [reply request-postgres/Dockerfile]: Decided to leave "as is" due to all other docker images

**Review by MantisClone — APPROVED** [review]

**Pull request merged**